### DISCUSSION TITLE
Import document in workflow build (pwd ?)

### DISCUSSION THREAD

**Discussion by thibaudbrg** [langgenius/dify#5156] [General]
Hi all,
Thanks for your awesome work.

I'm building a swagger for Confluence and I intend to use it into a workflow. To connect to my Confluence instance I have to give as parameter a certificate which is obviously a file. 

I'm trying to give the certificate that I have with me as a path to the variable I configured in my Confluence Swagger. But the question is, where am I to provide a path ? 

![image](https://github.com/langgenius/dify/assets/73235385/6929c4a8-b8e0-45f7-973a-8760021154d5)

I wish I could do `pwd` and know. 

I configured the docker-compose as you provided it in the repo. 
- Am I in the api container ? in the api-worker container ? in the web container ? 
I tried to put my certificate in all three containers but can't really find the connection. 
- Can I mount a volume in one of the containers to have a connection between Dify and my filesystem ? 
- Most efficiently, can I import a file directly into the workflow tab process from the Dify web app instead of having the need to bring it through a pathfile or volume mount via docker ? This would be the perfect scenario. A drag and drop possibility? 
![image](https://github.com/langgenius/dify/assets/73235385/6969412f-9202-440d-948e-264f691a22ea)

**Comment by ZTurboX** [comment]
@thibaudbrg have you build Confluence agent to dify?
